contrib/testChanges.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+cd `dirname $0`/..
+cargo build --locked --no-default-features --all
+cargo build --locked --no-default-features --all
+cargo build --locked --all
+cargo build --locked --features metrics_process --all
+cargo fmt
+cargo fmt --all -- --check
```

Comment:
If the above `cargo fmt` succeeds, this command will succeed too (since the code is already formatted)

Reply:
I'll remove it from the script then.